references/video_classification/presets.py

```diff
@@ -27,7 +27,7 @@ def __call__(self, x):
 
 
 class VideoClassificationPresetEval:
-    def __init__(self, resize_size, crop_size, mean=(0.43216, 0.394666, 0.37645), std=(0.22803, 0.22145, 0.216989)):
+    def __init__(self, crop_size, resize_size, mean=(0.43216, 0.394666, 0.37645), std=(0.22803, 0.22145, 0.216989)):
```

Comment:
Flipping their order to match the one of the presets in TorchVision. No BC concerns in references so that should be OK.